tests/FileTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Tests;
+
+use Rebilly\ApiKeyProvider;
+use Rebilly\Client;
+use Rebilly\Rest\File;
+use RuntimeException;
+
+/**
+ * Class FileTest.
+ *
+ * @group wip
+ */
+class FileTest extends TestCase
+{
+    public function testFile()
+    {
+        $client = new Client(['apiKey' => 'QWERTY']);
+
+        $client = new Client(
+            [
```

Comment:
also... why put this on a separate line... usually we used to put this in the line above like here for example - https://github.com/Rebilly/rebilly-php/blob/pdf/tests/ClientTest.php#L216